fabrictestbed/cli/cli.py

```diff
@@ -149,12 +166,10 @@ def query(ctx, cmhost: str, ochost: str, tokenlocation: str, projectid: str, sco
         if sliceid is None:
             status, response = slice_manager.slices(includes=includes)
         else:
-            slice_object = Slice()
-            slice_object.slice_id = sliceid
-            status, response = slice_manager.get_slice_topology(slice_object=slice_object)
+            status, response = slice_manager.slices(includes=includes,slice_id=sliceid)
```

Comment:
`slice_id` argument to SliceManager.slices() is allowed to be `None`, so I wonder if we need the if-else branches.

Reply:
Good catch - I was just following the existing logic (read hack) and missed that.